source/features/index.tsx

```diff
@@ -244,6 +244,18 @@ const add = async (id: FeatureID, ...loaders: FeatureLoader[]): Promise<void> =>
 	}
 };
 
+const addCssFeature = async (id: FeatureID, pageDetections: BooleanFunction[]): Promise<void> => {
+	void add(id, {
+		include: [
+			...pageDetections
```

Comment:
This variable should be called `include` and since it’s already an array you don’t need to use the rest parameter